doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -285,6 +285,7 @@ Bluetooth samples
 * :ref:`peripheral_fast_pair` sample:
 
   * Updated by disabling the :kconfig:option:`CONFIG_BT_SETTINGS_CCC_LAZY_LOADING` Kconfig option as a workaround fix for the `Zephyr issue #61033`_.
+  * Added automatic switching to the Fast Pair not discoverable advertising mode with the hide UI indication instead of removing Fast Pair advertising payload when all bond slots are taken.
```

Comment:
```suggestion
  * Added automatic switching to the Fast Pair not discoverable advertising mode with the hide UI indication instead of removing the Fast Pair advertising payload when all bond slots are taken.
```
We could interchange the 2 bullet point. Added and then updated.